public/docs/ts/latest/quickstart.jade

```diff
@@ -1,270 +1,505 @@
+      
+      * `npm run tsc:w` - run the TypeScript compiler in watch mode; 
+      the process keeps running, awaiting changes to TypeScript files and re-compiling when it sees them.
+      
+      * `npm run lite` - run the <a href="https://www.npmjs.com/package/lite-server" target="_blank">lite-server</a>, 
+      a light-weight, static file server, written and maintained by 
+      <a href="http://johnpapa.net/" target="_blank">John Papa</a> 
+      with excellent support for Angular apps that use routing.
+      
+      * `npm run typings` - runs the [*typings* tool](#typings)
+      
+      * `npm postinstall` - called by *npm* automatically *after* it successfully completes package installation.
+      This script installs the [TypeScript definition files](#typings) this app requires.
+      
+    button(onclick='why("add-package-json","package-json")' class='button button-small') Close
+      
+:marked
+  Open a terminal window (command window in Windows) and install these packages
+  with the npm command:
 code-example(format="").
   npm install
-:marked
-.l-sub-section
+
+.alert.is-important
   :marked
     Scary <span style="color:red; font-weight: bold">error messages in red</span> may appear **during** install. 
-    Ignore them. The install will succeed. See the [appendix below](#npm-errors) for more information.
+    The install typically <a href="javascript:why('npm-errors')">recovers</a> from these errors and finishes successfully.
+  #npm-errors(hidden)
+    :marked
+      #### *npm* errors and warnings
+      
+      All is well if there are no console messages starting with `npm ERR!` *at the end* of **npm install**. 
+      There might be a few `npm WARN` messages along the way &mdash; and that is perfectly fine.
+      
+      We often see an `npm WARN` message after a series of `gyp ERR!` messages. 
+      Ignore them. A package may try to re-compile itself using `node-gyp`.
+      If the re-compile fails, the package recovers (typically with a pre-built version)
+      and everything works.
+      
+      Just make sure there are no `npm ERR!` messages at the end of `npm install`.
+
 :marked
   **We're all set.** Let's write some code.
 
+
+a(id="component")
 .l-main-section
 :marked
   ## Our First Angular Component
-  
-  The *Component* is the most fundamental of Angular concepts. 
-  A component manages a view - a piece of the web page where we display information
-  to the user and respond to user feedback.
-  
-  Technically, a component is a class that controls a view template.
-  We'll write a lot of them as we build Angular apps. This is our first attempt
-  so we'll keep it ridiculously simple.
-  
-  ### Create an application source sub-folder
-  
-  We like to keep our application code in a sub-folder off the root called `app/`.
-  Execute the following command in the console window.
+button(onclick='why("what-is-a-component")' class='button button-small') Why
```

Comment:
This "why" needs to be preceded by an action. Maybe:

Let's create a super simple Angular component.

[WHY]